admin/controllers/attendance.js

```diff
@@ -243,16 +276,45 @@ const postDeclarationForm = async (req, res, next) => {
     })
   }
 
+  req.session.hdfFormData = form
+
   return res.redirect('/attendance/review-pupil-details')
 }
 
 const getHDFSubmitted = async (req, res, next) => {
-  res.locals.pageTitle = 'Headteacher\'s declaration form submitted'
+  res.locals.pageTitle = "Headteacher's declaration form"
+  req.breadcrumbs(res.locals.pageTitle)
+  try {
+    const hdf = await headteacherDeclarationService.findLatestHdfForSchool(req.user.School)
+    if (!hdf) {
+      return res.redirect('/attendance/declaration-form')
+    }
+    const resultsDate = hdf.checkEndDate.add(1, 'weeks').isoWeekday('Monday') // next monday after the check ends
+    const canViewResults = moment().isAfter(resultsDate)
```

Comment:
@danwilliams I am aware this is a two-liner but it concerns the presentation level rather than the controller so I would suggest adding a presenter. Thoughts?